flink-connector-oceanbase-cdc/src/main/java/com/ververica/cdc/connectors/oceanbase/OceanBaseSource.java

```diff
@@ -54,6 +54,7 @@ public static <T> Builder<T> builder() {
         private String tenantName;
         private String databaseName;
         private String tableName;
+        private String tableList;
```

Comment:
Hi @whhe, shall we use `databaseList` and `tableList` to keep the same form and the tableName field seems no longer needed.

https://github.com/ververica/flink-cdc-connectors/blob/02eca305d2682fcc0bfd626adb6db1d8b3213a9d/flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/source/config/MySqlSourceConfigFactory.java#L55-L56

Reply:
I noticed that MySqlSource is using `databaseList` and `tableList`, but I'm still not clear about why do we need a `databaseList` while `tableList` already contains the db name in every item. Is there something I missed?

Reply:
The `databaseList` and `tableList` are carried from debezium then other CDC connectors also keep the same form.

| Property | Description  |
|:-:|:-:|
|database.include.list | An optional, comma-separated list of regular expressions that match the names of the databases for which to capture changes. The connector does not capture changes in any database whose name is not in database.include.list. By default, the connector captures changes in all databases. Do not also set the database.exclude.list connector confiuration property. |
| table.include.list | An optional, comma-separated list of regular expressions that match fully-qualified table identifiers of tables whose changes you want to capture. The connector does not capture changes in any table not included in table.include.list. Each identifier is of the form databaseName.tableName. By default, the connector captures changes in every non-system table in each database whose changes are being captured. Do not also specify the table.exclude.list connector configuration property. |

Please refer to https://debezium.io/documentation/reference/1.6/connectors/mysql.htm.

Reply:
The database(s) and table(s) regex feature is mostly used in the stream api. 
The `databaseList` or `tableList` can be used independently as described above.

Reply:
So `databaseList` is used to sync the whole database in the list? If it's ture, then `databaseList` might be similar with `table-name` regex in usage. If we keep the options like `database-name` and `table-name`, `databaseList` would be unnecessary.